server/views/pages/about-not-complete.njk

```diff
@@ -0,0 +1,107 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
+{% from "moj/components/button-menu/macro.njk" import mojButtonMenu %}
+{%- from "moj/components/banner/macro.njk" import mojBanner -%}
+{% from "../components/assessment/assessment.njk" import renderAssessment %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set print = true %}
+{% set locale = interpolate(locale, {
+    subject: data.popData,
+    lastUpdatedDate: data.assessmentAreas.versionUpdatedAt | formatSimpleDate
+}) %}
+
+{% block pageTitle %}
+    {% if not errors %}
+        {{ locale.page.title }}
+    {% else %}
+        {{ locale.page.errorTitle }}
+    {% endif %}
+{% endblock %}
+
+{% set mainClasses = "app-container govuk-body" %}
+{% set header = {
+    type: "plan-extended",
+    heading: locale.mainHeading.title,
+    items: [
+        {
+            text: locale.common.header.returnToOasysButton,
+            href: data.oasysReturnUrl,
+            classes: "govuk-button--secondary",
+            type: 'button'
+        }
+    ]
+} %}
+
+{% block bodyStart %}
+    <div id="top"></div>
+{% endblock %}
+
+{% block content %}
+    {{ super() }}
+    {% if data.assessmentAreas.versionUpdatedAt %}
+        <p class="govuk-body govuk-!-margin-top-5">{{ locale.detail.lastUpdated }}</p>
+    {% endif %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {% if errors.domain %}
+            {%  set errorList = [] %}
+            {%  for error in errors.domain %}
+                {%  set errorList = (errorList.push({ text: locale.errors[error]}), errorList)    %}
+            {%  endfor %}
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: errorList
+                }) }}
+            {% endif %}
+
+            {% if data.deliusData.sentences.length > 0 %}
```

Comment:
is this identical to the section in about.njk? if so, can we extract it and include it instead of duplicating?

Reply:
OK - can you add a subtask to the Jira story which has the "real" solution on it to delete this page please?

Reply:
I'll sort this out 👍